Front-end/glaucoma-diagnosis-system/src/views/adminManage.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+        <div>
+            <div class="flex mt-3 ml-3">
+                <ul class="flex border-b">
+  <li class="-mb-px mr-1">
```

Comment:
Please do Right click -> Format :), till we setup eslint

Reply:
comments addressed